pilot/pkg/trustbundle/trustbundle_test.go

```diff
@@ -0,0 +1,135 @@
+// Copyright Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package trustbundle
+
+import (
+	"io/ioutil"
+	"path"
+	"sort"
+	"testing"
+
+	"istio.io/istio/pkg/test/env"
+)
+
+func readCertFromFile(filename string) string {
+	csrBytes, err := ioutil.ReadFile(filename)
+	if err != nil {
+		return ""
+	}
+	return string(csrBytes)
+}
+
+var testCerts map[string]string = map[string]string{
```

Comment:
Nit: a map here seems to just make things untyped. Would it be more explicit to just use 4 variables? Or a struct if you want them grouped + typed